src/mpid/ch4/netmod/ucx/ucx_init.c

```diff
@@ -249,6 +249,11 @@ int MPIDI_UCX_mpi_init_hook(int rank, int size, int appnum, int *tag_bits, MPIR_
     mpi_errno = initial_address_exchange(init_comm);
     MPIR_ERR_CHECK(mpi_errno);
 
+    ucx_status =
+        ucp_worker_set_am_handler(MPIDI_UCX_global.ctx[0].worker, 0, &MPIDI_UCX_am_handler, NULL,
+                                  UCP_AM_FLAG_WHOLE_MSG);
```

Comment:
The `0`, the second argument seems too magical, how about define a macro and use the macro instead?

(I know that the `0` in the `ctx` is also magical, but that is temporary until multiple vci gets implemented.)

Reply:
Sure. Let's see where the tests are after the next run. I saw a bunch of ubsan stuff that probably needs to be addressed. 

If we are to go this direction, we could actually register handlers for all MPIDIG packet types and eliminate the ucx header added by the netmod, but that is getting a few steps ahead.

Reply:
Fixed.